docs/UsersGuide/best/hub-pattern.md

```diff
@@ -16,5 +16,132 @@ ARINC 653 Channels, High-speed hardware buses between nodes, and UARTs between n
+excluded from this example for simplicity, a `Fw::ComStub` and `Fw::ComQueue` components could be 
+added to make this communication link between hubs more robust. The Hub uses the `Data Out` port to 
+export serialized data to the remote hub, which receives them on the `Data In` port. This connection 
+works reflexively, allowing the two hubs to communicate and transmit data between each other. 
+
+A [Command Splitter](../api/c++/html/svc_cmd_splitter.html) sits between the Ground Interface and a 
+local [Command Dispatcher](../api/c++/html/svc_cmd_dispatcher_component.html) and splits command 
+opcodes to either a local or remote command dispatcher. The command opcode offset of the command 
+splitter should match the commands on the external node. An offset of 0x10000 will route any command 
+received above the offset directly to the hub, while commands below the offset will be routed to the 
+local Command Dispatcher.Instance ids must also match this offset, which may be implemented as the 
+following for each instance in instances.fpp:
+```shell
+# In Node B's instances.fpp
+module NodeBDeployment {
+  constant CMD_SPLITTER_OFFSET = 0x10000
+  ...
+  instance b_hub: Svc.GenericHub base id CMD_SPLITTER_OFFSET + 0x9000
+  ...
+}
+```
+> Configure the Command Splitter by calling `cmdSplitter.configure(0x10000);` in Node A's topology.cpp
+
+
+`Port In` and `Port Out` are arrays of serial ports that should be parallel to each other across 
+hubs. A connection to Hub A's `Port In` index 0 will be routed to Hub B's `Port Out` index 0. In 
+this example, the Command Splitter in Node A is connected to the Command Dispatcher in Node B 
+through the hub port arrays. Data flow from Node B to A work in the same way, such that Hub B's 
+`Port In` will be parallel to Hub A's `Port Out`.
+
+To receive telemetry from Node B through the Ground Interface from Node A, Telemetry and Event 
+Connections can be defined through the pattern graph specifiers. The hub replaces 
+`Svc::ActiveLogger` and `Svc::TlmChan` in the pattern graph specifiers of Node B, which would 
+reroute all events and logs to the hub and to the ActiveLogger and TlmChan of Node A. Events and 
+Telemetry have a dedicated `Port In` the hub, which are connected to Node A's Event Logger and Tlm 
+Send through the Log Send and Tlm Send port of the hub.
+
+```shell
+  # In Node B's topology.fpp  
+    ...
+    # event connections instance b_eventLogger
+    event connections instance b_hub 
+
+    # telemetry connections instance b_tlmSend
+    telemetry connections instance b_hub
+    ...
+```
+```shell
+  # In Node A's topology.fpp  
+    ...
+    connections hub {
+      a_hub.LogSend -> a_eventLogger.LogRecv
+      a_hub.TlmSend -> a_tlmSend.TlmRecv
+    } 
+    ...
+```
+
+### Implementation Notes
+All instances names should be unique across the entire F´ System. Instances with the same name may 
```

Comment:
What about something like: Telemetry channels and events are currently named based on the component instance name, so re-using the same instance name between multiple deployments can result in naming conflicts.